testfiles/SurfacePropTest_SurfLWR.idf

```diff
@@ -2303,7 +2302,8 @@
     Zn005:Wall002:Win001,    !- Exterior Surface Name
     ,                        !- External Shading Fraction Schedule Name
     SrdSurfs:Zn005:Wall002:Win001,  !- Surrounding Surfaces Object Name
-    ;                        !- Outdoor Air Node Name
+    ,                        !- Outdoor Air Node Name
+    GndSurfs:Zn005:Wall002:Win001;  !- Ground Surfaces Object Name
```

Comment:
This is manual transition. Added the name of the new object to the new field added to `SurfaceProperty:LocalEnvironment` object.